lib/restaurantInfo.js

```diff
@@ -0,0 +1,17 @@
+const { collection } = require("./database");
+
+async function getRandomRestaurant(cuisine_value) {
```

Comment:
Maybe use camelCase (`cuisineValue`)?

Reply:
This is a good suggestion! i'll change it when i am refactore my code next week 🦾 thanks a lot